modules/peers.js

```diff
@@ -545,7 +626,7 @@ Peers.prototype.onPeersReady = function () {
 		});
 	}
 	// Loop in 10sec intervals (5sec + 5sec connect timeout from pingPeer)
-	jobsQueue.register('peersDiscoveryAndUpdate', peersDiscoveryAndUpdate, 5000);
+	jobsQueue.register('peersDiscoveryAndUpdate', peersDiscoveryAndUpdate, 10000);
```

Comment:
I agree. We receive updates from peers regularly and therefore asking them for state may not be as neccessary as it was before. 60 seconds should be sufficient, and perhaps we could check only peers we havent heard from recently instead of all of our peers.